doc/source/user_guide/tutorials/plot/plotting_data_on_geometry_elements.rst

```diff
@@ -0,0 +1,388 @@
+.. _ref_plotting_data_on_geometry_elements:
+
+==============================
+Plot data on geometry elements
+==============================
+
+.. |add_mesh| replace:: :func:`add_mesh()<ansys.dpf.core.plotter.DpfPlotter.add_mesh>`
+.. |add_field| replace:: :func:`add_field()<ansys.dpf.core.plotter.DpfPlotter.add_field>`
+.. |show_figure| replace:: :func:`show_figure()<ansys.dpf.core.plotter.DpfPlotter.show_figure>`
+.. |Model| replace:: :class:`Model <ansys.dpf.core.model.Model>`
+.. |Line| replace:: :class:`Line <ansys.dpf.core.geometry.Line>`
+.. |Points| replace:: :class:`Points <ansys.dpf.core.geometry.Points>`
+.. |Plane| replace:: :class:`Plane <ansys.dpf.core.geometry.Plane>`
+.. |mapping| replace:: :class:`mapping <ansys.dpf.core.operators.mapping.on_coordinates.on_coordinates>`
+.. |nodes_coordinates| replace:: :class:`nodes_coordinates<ansys.dpf.core.operators.mesh.node_coordinates.node_coordinates>`
+.. |Points.plot| replace:: :func:`Points.plot()<ansys.dpf.core.geometry.Points.plot>`
+.. |Line.plot| replace:: :func:`Line.plot()<ansys.dpf.core.geometry.Line.plot>`
+.. |Plane.plot| replace:: :func:`Plane.plot()<ansys.dpf.core.geometry.Plane.plot>`
+
+This tutorials shows how to get a result mapped over different geometric objects:
+
+- Points
+- Line
+- Plane
+
+Define the data
+---------------
+
+First, import a results file. For this tutorial, you can use the one available in the |Examples| module.
+For more information about how to import your own result file in DPF, see
+the :ref:`ref_tutorials_import_data` tutorials section.
+
+.. jupyter-execute::
+
+    # Import the ``ansys.dpf.core`` module
+    from ansys.dpf import core as dpf
+    # Import the examples module
+    from ansys.dpf.core import examples
+    # Import the operators module
+    from ansys.dpf.core import operators as ops
+    # Import the geometry module
+    from ansys.dpf.core import geometry as geo
+
+    # Define the result file path
+    result_file_path_1 = examples.find_static_rst()
```

Comment:
@luisaFelixSalles in this tutorial everything is marked as `*_1` whereas you used to do `my_*` in other tutorials? I that the new guideline for variable names in tutorials?